ui/src/app/applications/components/application-pod-view/pod-view.tsx

```diff
@@ -167,13 +167,20 @@ export class PodView extends React.Component<PodViewProps> {
                                                                                 <div>
                                                                                     {pod.metadata.name}
                                                                                     <div>Health: {pod.health}</div>
+                                                                                    <div>
```

Comment:
While I agree we should display this information in the pod view, I think in the current format it's information overload. I'd like to have a better design for the tooltip (or pod view in general) for a separate PR.

Reply:
Updated the PR, extracting this tooltip to a separate component (so that we can properly style it in a future PR).

*Before*:

<img width="355" alt="Screenshot 2023-02-22 at 03 03 37" src="https://user-images.githubusercontent.com/346576/220507725-e54488b7-96ff-4089-ac56-3045ae4c7c3c.png">

*After*:

<img width="333" alt="Screenshot 2023-02-22 at 03 42 18" src="https://user-images.githubusercontent.com/346576/220507759-fa844aba-0b5a-49e4-8b2f-e76cacfa0b62.png">

Reply:
Looks great!